LEGO1/legoworld.cpp

```diff
@@ -26,12 +34,35 @@ void LegoWorld::Stop()
 	TickleManager()->UnregisterClient(this);
 }
 
+// OFFSET: LEGO1 0x1001f5e0
+MxLong LegoWorld::Notify(MxParam& p_param)
```

Comment:
This *may* be an MxResult though I think other :Notify functions are MxLong's, so check on that

Reply:
`MxResult` is a typedef for `MxLong`. However, unless we have enough evidence that all Notify functions restrict themselves to returning one of the two `MxResult` constants, we should leave it as-is